presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -252,6 +255,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         STATISTICS_ENABLED,
                         "Experimental: Expose table statistics",
                         hiveClientConfig.isTableStatisticsEnabled(),
+                        false),
+                integerProperty(
+                        PARTITION_STATISTICS_SAMPLE_SIZE,
+                        "Maximum sample size of the partitions column statistics",
```

Comment:
add `Experimental` because above property is also experimental

Reply:
ping-pong. https://github.com/prestodb/presto/pull/10960#discussion_r201924805